docs/source/technical_features.md

```diff
@@ -51,11 +36,11 @@ the prior knowledge about the subjective lesion borders is then leveraged to the
 
 ![Figure Data Augmentation on lesion ground truths](https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/dilate-gt.png)
 
-### Network architectures
+## Network architectures
 - [U-net](https://arxiv.org/pdf/1505.04597.pdf)
```

Comment:
Maybe also reference what files these implementations are in?

Reply:
what do you mean by "what files"? do you mean that we should add a link that points to the actual files in the code that implement these models? If so, this is out of the scope of this PR (if that's what you meant i'll open an issue)

Reply:
Ya, that is what I meant. I don't know if it needs to be done, just throwing out the idea.

Reply:
great idea-- issue opened https://github.com/neuropoly/ivado-medical-imaging/issues/270